tests/cdc-bank/bank.go

```diff
@@ -214,15 +216,22 @@ func (c *client) Start(ctx context.Context, cfg interface{}, clientNodes []clust
 						_ = txn.Rollback()
 						continue
 					}
-					if balanceSum != expectedSum {
-						log.Fatalf("[cdc-bank] [validatorId=%d] sum: %d, expected: %d, startTS: %d", idx, balanceSum, expectedSum, startTS)
+					if balanceSum == expectedSum {
+						atomic.AddInt32(&counter, 1)
+						_ = txn.Rollback()
+						log.Infof("[cdc-bank] [validatorId=%d] success, startTS: %d", idx, startTS)
+						return
```

Comment:
It breaks the original test logical. We want to repeatedly check the total balance during the test (before the `ctx` done), but now it only check `validateConcurrency` times at the beginning.